packages/zowe-explorer/CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 - Fixed an issue where the 'Delete' key binding for the USS tree returns a 'contextValue' error. [#2796](https://github.com/zowe/zowe-explorer-vscode/issues/2796)
 - Fixed an issue where cancelling Unix, MVS or TSO command still submits the command. [#3422](https://github.com/zowe/zowe-explorer-vscode/issues/3422)
 - Fixed an issue where user is unable to open a renamed sequential data set from the Data Sets tree view.. [#3345](https://github.com/zowe/zowe-explorer-vscode/issues/3345)
+- Fixed an issue seen with outdated profile information in the z/OS tree view data during upload and download of dataset and uss files
```

Comment:
- "data set" is two words
- "uss" should be capitalized

Reply:
resolved in 1e28ad60601f9f638578c1286850541472c9a415